### ISSUE TITLE
Navigation regions lack aria-labels to clarify their specific purpose

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-book#874]
### Description

When multiple regions of the same type exist on a page, each should have a unique label to identify its purpose. Currently there are three navigation regions within the Pressbooks reading interface (main, contents, and next/previous chapter), but their individual purpose is not identified. 
![Screenshot from 2021-10-12 15-12-17](https://user-images.githubusercontent.com/13485451/137035989-da75e34f-0585-47e8-be49-a7918c5227bf.png)

We should add an aria-label to each region to clarify the specific purpose of each navigation section.

**Comment by SteelWagstaff** [comment]
Tested and confirmed working on integrations.
